mainhandler/handlerequests.go

```diff
@@ -85,7 +87,15 @@ func (mainHandler *MainHandler) HandleWatchers(ctx context.Context) {
 		}
 	}()
 
-	watchHandler, err := watcher.NewWatchHandler(ctx)
+	config, err := rest.InClusterConfig()
+	if err != nil {
+		panic(err)
+	}
+	ksStorageClient, err := kssc.NewForConfig(config)
```

Comment:
Here you can call the `k8sinterface.GetK8sConfig()` function, this is what I expect to see here:
```suggestion
	ksStorageClient, err := kssc.NewForConfig(k8sinterface.GetK8sConfig())
```
The `rest.InClusterConfig()` can be removed

Reply:
Addressed in the latest commit.